simulator/config.py

```diff
@@ -1,11 +1,18 @@
-""" Defines various color constants. """
-
-# Cube colors.
-CUBE_RED = "#DB4D67"
-CUBE_BLUE = "#146687"
-CUBE_GOLD = "#87821B"
-
-# Default screen color.
-SCREEN = "#35A6D4"
-# Button color.
-BUTTONS = "#051B24"
+import logger
+
+# Base cube size, in px.
+CUBE_SIZE = 200
+
+log = logger.Logger()

```

Comment:
This is actually not the most convenient way to do this. Have a look at [logging.setLoggerClass()](https://docs.python.org/2/library/logging.html#logging.setLoggerClass) and [logging.getLogger()](https://docs.python.org/2/library/logging.html#logging.getLogger).